spec/tests/manual.fmf

```diff
@@ -1,50 +1,32 @@
-summary: Manual test execution details
+summary: Test automation state
 
 story:
-    As a tester I want to store detailed manual instructions
-    covering test scenario so that I can perform them manually
-    before the test can be fully automated.
+    As a tester I need to store detailed manual instructions covering
+    test scenario I have to perform manually.
 
 description: |
-    Sometimes test scenarios cannot be fully automated or it takes
-    time to fully automate them. For such cases the ``manual``
-    attribute can be used to describe test steps in detail. It
-    should be a ``dictionary`` with the following keys, each one
-    of them should be a ``string``.
-
-    setup
-        Environment preparation instructions.
-    step
-        Test step, action to be performed.
-    expect
-        Expected results of the test step.
-    cleanup
-        Environment cleanup instructions.
-
-    If there are several individual steps they can be stored, with
-    their corresponding expected results, as a list of dictionaries
-    under the ``steps`` key.
+    Attribute marks whether this test needs human interaction during its execution.
+
+    Such tests are not likely to be executed in automation pipelines.
+    In the future they can be executed in semi-automated way, waiting on human interaction.
+
+    It's value should be a ``bool``. The default value is ``false``.
+
+    Attribute :ref:`spec/tests/test` should point to a Markdown document in defined format.
```

Comment:
```suggestion
    Attribute :ref:`/spec/tests/test` should point to a Markdown document in defined format.
```

Reply:
It should be a full fmf id, otherwise the ref does not work. You can check the result using `make docs`.